### PR TITLE
[ADD] Syncplay 1.6.4a

### PR DESCRIPTION
**Pull request by anodium**
Client/server to synchronize media playback on mpv/VLC/MPC-HC/MPC-BE on many computers

https://syncplay.pl/

### CODE REVIEW

File: bucket/syncplay.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    "extract_to": "",
+    "persist": [
+        "syncplay.ini"
+    ],
+    "checkver": {
+        "github": "https://github.com/Syncplay/syncplay"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Syncplay/syncplay/releases/download/$version/Syncplay_$version_Portable.exe"
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
        "url": "https://github.com/Syncplay/syncplay/releases/download/v$version/Syncplay_$version_Portable.zip"
```

File: bucket/syncplay.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "notes": [
+        "Remember to install a supported media player!",
+        "mpv is the most well-supported."
+    ],
+    "extract_to": "",
+    "persist": [
+        "syncplay.ini"
+    ],
+    "checkver": {
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
    "checkver": ">Syncplay ([\\w.]+)<",
```

File: bucket/syncplay.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://syncplay.pl/",
+    "description": "Client/server to synchronize media playback on mpv/VLC/MPC-HC/MPC-BE on many computers",
+    "license": "Apache-2.0",
+    "version": "1.6.4a",
+    "url": "https://github.com/Syncplay/syncplay/releases/download/v1.6.4a/Syncplay_1.6.4a_Portable.zip",
+    "hash": "sha512:38d37268d3d1d9dadec112ec5fa9db92519e1ed0e7f104bcb6cdda594e7bc26da73359ed418bb1a97337b87fbfa80cae5c4860d8e29af694d539ce3241581d64",
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
    "hash": "40bf01758a561c1130664b7af3e6c50ed45a3661bd7be552a13faae401a84ef0",
```

File: bucket/syncplay.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "bin": ["Syncplay.exe", "syncplayServer.exe"],
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
    "bin": [
        "Syncplay.exe",
        "syncplayServer.exe"
    ],
    "shortcuts": [
        [
            "Syncplay.exe",
            "Syncplay"
        ]
    ],
```

File: bucket/syncplay.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        "vlc": "extras/vlc",
+        "mpc-be": "extras/mpc-be",
+        "mpc-hc": "extras/mpc-hc",
+        "mpc-hc-fork": "extras/mpc-hc-fork"
+    },
+    "notes": [
+        "Remember to install a supported media player!",
+        "mpv is the most well-supported."
+    ],
+    "extract_to": "",
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
```

File: bucket/syncplay.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    "persist": [
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
    "persist": "syncplay.ini",
```

**Comment by linsui** [comment]
Why depends on youtube-dl? Is there any information about this?

File: bucket/syncplay.json [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    "homepage": "https://syncplay.pl/",
+    "description": "Client/server to synchronize media playback on mpv/VLC/MPC-HC/MPC-BE on many computers",
+    "license": "Apache-2.0",
+    "version": "1.6.4a",
+    "url": "https://github.com/Syncplay/syncplay/releases/download/v1.6.4a/Syncplay_1.6.4a_Portable.zip",
+    "hash": "sha512:38d37268d3d1d9dadec112ec5fa9db92519e1ed0e7f104bcb6cdda594e7bc26da73359ed418bb1a97337b87fbfa80cae5c4860d8e29af694d539ce3241581d64",
+    "bin": ["Syncplay.exe", "syncplayServer.exe"],
+    "depends": "youtube-dl",
+    "suggest": {
+        "mpv": "extras/mpv",
```
Comment (linsui) [inline bucket/syncplay.json]:
```suggestion
        "Vedio Player": [
            "extras/mpv",
            "extras/mpv-git",
            "extras/vlc",
            "extras/mpc-be",
            "extras/mpc-hc",
            "extras/mpc-hc-fork"
        ]
```

**Comment by pratikpc** [comment]
Closing in favour of https://github.com/ScoopInstaller/Extras/pull/6824

**Pull request closed without merging**